geoengine/__init__.py

```diff
@@ -8,3 +11,12 @@
 from .workflow import WorkflowId, Workflow, workflow_by_id, register_workflow
 from .datasets import upload_dataframe, StoredDataset
 from .colorizer import Colorizer
+
+DEFAULT_USER_AGENT = f'geoengine-python/{get_distribution("geoengine").version}'
+
+
+def default_user_agent(_name="python-requests"):
+    return DEFAULT_USER_AGENT
+
+
+utils.default_user_agent = default_user_agent
```

Comment:
Can we add a test for that or is that difficult?
Maybe with contains?

Reply:
1ac93acd44c17e69741e5440a59e7b1ed5b756f7